### ISSUE TITLE
Precompute Ys fails when r requires gradients 

### ISSUE THREAD

**Issue by bkmi** [mariogeiger/se3cnn#26]
https://github.com/mariogeiger/se3cnn/blob/bbe5871f41317b34904706f7198b8508b1a6d1d6/se3cnn/point/kernel.py#L116

Since we are getting our Ys from numpy, there is a problem when this line is called and r requires a gradient. I think the fix of ```r.cpu().detach()``` is enough, but I'm not sure if it breaks anything. I have made this change in my point branch and will add it.

**Comment by mariogeiger** [comment]
this is not a bug

**Comment by blondegeek** [comment]
To add to this, in order to have gradients to r you typically need to use
backwardable spherical harmonics. If you only want to have gradients go
through the radial function then detaching is something you can do —
however in doing so you are deliberately neglecting gradient contributions
from the angular portion of the kernel. Does that make sense?
